sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -1120,39 +1139,23 @@ def test_warm_start_wo_nestimators_change():
     assert clf.estimators_.shape[0] == 10
 
 
-def test_probability_exponential(global_random_seed):
-    # Predict probabilities.
-    clf = GradientBoostingClassifier(
-        loss="exponential", n_estimators=100, random_state=global_random_seed
-    )
-
-    with pytest.raises(ValueError):
-        clf.predict_proba(T)
-
-    clf.fit(X, y)
-    assert_array_equal(clf.predict(T), true_result)
-
-    # check if probabilities are in [0, 1].
-    y_proba = clf.predict_proba(T)
-    assert np.all(y_proba >= 0.0)
-    assert np.all(y_proba <= 1.0)
-    score = clf.decision_function(T).ravel()
-    assert_allclose(y_proba[:, 1], expit(2 * score))
-
-    # derive predictions from probabilities
-    y_pred = clf.classes_.take(y_proba.argmax(axis=1), axis=0)
-    assert_array_equal(y_pred, true_result)
-
-
-def test_non_uniform_weights_toy_edge_case_reg():
+@pytest.mark.parametrize(
+    ("loss", "value"),
+    [
+        ("squared_error", 0.5),
+        ("absolute_error", 0.0),
```

Comment:
"absolute_error" still has a value of 0.0. If this is required for this particular loss, then it seems fine. Otherwise why not just remove "value" from the parametrization and set `gb.predict([[1, 0]])[0] >= 0.5`?

Reply:
That's needed. I tested 0, but it does not work.

Reply:
I guess that was the reason I introduces the parametrization with loss and value.

Reply:
Sounds good. Thank you for clarifying.

Reply:
However we also do not have a backward compatibility test for absolute error. Would it make sense to add that?

Reply:
If you insist, then yes. It'll take some time for me to do that.

Reply:
No I don't insist 😄 . I think there are other tests that aren't changed, which do use `abolute_error`, so hopefully that should prove that the results are consistent.

Reply:
Specifically this one
https://github.com/scikit-learn/scikit-learn/blob/51ca717db5065e912cdd6cd838bb08d4d08c2770/sklearn/ensemble/tests/test_gradient_boosting.py#L120-L158